src/components/BookList.js

```diff
@@ -1,23 +1,41 @@
+/* eslint-disable no-unused-vars */
 /* eslint-disable no-console */
-/* eslint-disable array-callback-return */
 /* eslint-disable react/destructuring-assignment */
 /* eslint-disable react/prop-types */
+/* eslint-disable no-alert */
+/* eslint-disable no-unused-expressions */
```

Comment:
- [ ] You have disabled too many linters, Linters help us to write clean code, please try to fix those errors  instead of disabling them, I am giving this as required changes because you disabled many, kindly reduce the number of linters rule you disable